docs/goals.md

```diff
@@ -12,43 +12,43 @@ of some data value or values.
 
 The design goals are listed below.
 
-1.  Express grammatical features, such as plurals, genders, and inflections.
+1.  Express grammatical features, such as plurals and genders.
```

Comment:
We aren't supporting genders in this release. Either we should remove that too or we can keep inflections.

Reply:
Let's remove then. Only a "such as"

Reply:
```suggestion
1. Express grammatical features. For example: plurals.
```